_posts/2018-7-18-EIA-coal.md

```diff
@@ -0,0 +1,141 @@
+
+This technique allows us to visually organize the shipments between diverse regions, with the width of the lines representing the size of the shipments in millions of tons.  The axes show the total amount of coal produced and consumed within that year.  Arrows looping back to the same region indicate coal produced and consumed in the same region.
+
+To avoid the visualization from being overly cluttered, we group the US states based on [US Census Divisions](https://en.wikipedia.org/wiki/List_of_regions_of_the_United_States#/media/File:Census_Regions_and_Division_of_the_United_States.svg){:target="_blank"}, with the abbreviations for states in each division indicated.
+
+![Chord Diagram]({{ site.baseurl }}public/images/YearlyRegionalCoalFlows.gif)
+<sup>_Yearly coal flows between different US Census Divisions._</sup>
+
+In the three divisions at the top of the plot (_West North Central_, _West South Central_, and _East North Central_), the majority of coal is sourced from states in the Mountain division.  The locations of the top five US coal producing states on the plot are indicated below.  This list uses [statistics from the EIA for 2016](https://www.eia.gov/tools/faqs/faq.php?id=69&t=2){:target="_blank"} and includes the total amount of production in megatons, along with their percent contribution to overall US production:
+
+* **Wyoming**: 297.2 MT (41%) - _Mountain (bottom)_
+* **West Virginia**: 79.8 MT (11%) - _South Atlantic (left)_
+* **Pennsylvania**: 45.7 MT (6%) - _Middle Atlantic (right, below middle)_
+* **Illinois**: 43.4 MT (6%) - _East North Central (top right)_
+* **Kentucky**: 42.9 MT (6%) - _East South Central (right, above middle)_
+
+Overall, what we can see is that coal shipments have been steading decreasing since a peak around 2010.  Most of the different regions are not self-sufficient, with shipments between regions being common.  Only _Mountain_ is self-sufficient and also serves as the dominant supplier in other regions as well.  Looking a bit deeper, checking the [annual coal production statistics for the Powder River Basin](https://www.eia.gov/opendata/qb.php?category=717304&sdid=COAL.PRODUCTION.TOT-PRB-TOT.A){:target="_blank"} reveals that with between 313 and 495 MT of annual shipments, it's the single area responsible for the vast majority of coal shipments originating from _Mountain_.
+
+# Coal Consumption
+
+We now look at what happens to the coal once its used for electricity generation, and also put this in context of total electricity generation from all fuel sources.  For this we use the [bulk electricity data](http://api.eia.gov/bulk/ELEC.zip){:target="_blank"}, specifically the [plant level data which can be browsed online](https://www.eia.gov/opendata/qb.php?category=1017){:target="_blank"}.  This data contains monthly information on each power plant with statistics on the fuel types, amount of electricity generation and fuel consumed.
```

Comment:
--> "This data contains monthly information on each power plant with statistics on the fuel types, amount of electricity generation, and fuel consumed."
(oxford comma ;) )